### PR TITLE
docs: add Ankit to GDE resources

### PR DESCRIPTION
**Pull request by AnkitSharma-007**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -100,6 +100,14 @@
     "bio": "Natalia started working professionally as a web software developer in the early 2000s. She now works in the role of technical lead and software/solutions architect. She is a public speaker, content creator, GDE for Angular and Web Technologies, Microsoft MVP Award (Developer Technologies), and a mentor with the RoadtoGDE program.",
     "groups": ["GDE"]
   },
+  "AnkitSharma-007": {
+    "name": "Ankit",
```
Comment (twerske) [inline aio/content/marketing/contributors.json:104]: Would you be able to include your full first name?

**Review by twerske — APPROVED** [review]
One comment before merging but lgtm!

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (AnkitSharma-007) [reply aio/content/marketing/contributors.json]: I have added the full name as per your comment.

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit ccb73c0ac8228c4c3f07ce8c63e17fc48ab4b483.

**Pull request closed without merging**